client/platform/desktop/backend/native/multiCamUtils.ts

```diff
@@ -14,22 +14,29 @@ function transcodeMultiCam(
 ) {
   let destLoc = '';
   if (jsonMeta.multiCam) {
+    let breakFlag = false;
     Object.entries(jsonMeta.multiCam.cameras).forEach(([key, val]) => {
```

Comment:
This can be done but ESLint gets upset and requires `// eslint-disable-next-line no-restricted-syntax` mostly because it doesn't want you to use a for of like that.  I think the `for...of` is cleaner than the break flag I was using which was causing additional iterations.

Reply:
I also usually forget that for...of and for...in require RegeneratorRuntime, which I avoid like the plague.

You might just need a regular for loop here?